source/developers-guide/shopware-composer/index.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: default
+title: Using Composer to install Shopware
+github_link: developers-guide/shopware-composer/index.md
+indexed: true
+group: Developer Guides
+subgroup: General Resources
+menu_title: Using composer with Shopware
+menu_order: 45
+---
+
+<div class="toc-list"></div>
+
+## Introduction
+
+Starting with v5.4 Shopware supports installing a shop as a composer dependency out of the box. This helps you to
+professionalize development and deployments of Shopware shops by providing a reliable versioning of Shopware itself and
+all the plugins required by your project.
+
+## What is composer
+
+Composer defines itself as "[...] a tool for dependency management in PHP. It allows you to declare the libraries
+your project depends on and it will manage (install/update) them for you" ([Composer website](https://getcomposer.org/)).
+
+Composer is the de-facto-standard for dependency management in the PHP community. You can think of it as the
+apt/yum/npm/brew of the PHP world.
+
+## How to start
+
+To create a new Shopware project with Shopware as a dependency, all you need to do is run
+
+```bash
+composer create-project shopware/composer-project my_project_name --no-interaction --stability=dev
+```
+
+This will clone the project repository with all necessary dependencies (including the latest released Shopware version)
+into a new directory `my_project_name`. Shopware itself will be deployed under `vendor/shopware/shopware` inside of this
+project directory, but it won't write inside of there.
+
+## Configuring Shopware
+
+The Composer Shopware project template relies on environment variables to configure your Shopware project. You can
+either set those directly (this is recommended on production environments e.g. to not store credentials on disk) or
+using a `.env` file in the project root. To see which variables are supported, have a look at the `.env.example` file.
+
+You can also have a `.env` file created by you! Simply run `app/install.sh` inside your new project directory to have a
+little installer-script ask you all necessary information.
```

Comment:
You can also create an own `.env` file! Simply run `app/install.sh` inside your new project directory to start a little installer-script asking you all necessary information.

Reply:
Wouldn't this sound better?
"You can also have a `.env` file created for you!"

Reply:
yes, that's ok